Sources/SecureXPC/XPCServiceDescriptor.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  XPCServiceDescriptor.swift
+//  
+//
+//  Created by Alexander Momchilov on 2021-11-28.
+//
+
+import Foundation
+
+public enum XPCServiceDescriptor {
```

Comment:
Does this need to be `public`? It's usage in `XPCServerEndpoint` is internal only.